bgpd/bgp_route.c

```diff
@@ -4218,10 +4218,6 @@ void bgp_update(struct peer *peer, const struct prefix *p, uint32_t addpath_id,
 	if (orig_safi == SAFI_LABELED_UNICAST)
 		safi = SAFI_UNICAST;
 
-	memset(&new_attr, 0, sizeof(new_attr));
```

Comment:
I think we should initialize at least it at the declaration `struct attr new_attr = {};`, to avoid some tricky edge cases? (don't know if coverity does not complain too)

Reply:
It won't change anything